proof-of-file-existence/tutorial/02-pofe-chain.md

```diff
@@ -0,0 +1,157 @@
+---
+order: 1
+---
+
+
+# Building our PoFE blockchain using Starport
+
+We can scaffold our application by running `starport app github.com/user/pofe`. This creates a new folder called `pofe` which contains the code for your app.
+
+```
+starport app github.com/user/pofe
+```
+
+## Run application
+
+After scaffolding our application, let's open up a separate terminal window in the newly created `pofe` folder and run `starport serve`, which will start our application. Here, you should be able to see the following output, as well as any errors that show up in your application.
+
+```
+$ starport serve
+Cosmos' version is: Launchpad
+
+📦 Installing dependencies...
+🛠️  Building the app...
+🙂 Created an account. Password (mnemonic): joy manage firm arrange finish bounce power shove bring bundle issue chief quiz spread symptom sword slender vote hobby water weird trim panther slot
+🙂 Created an account. Password (mnemonic): meat skirt slab smart impulse region clump genuine zero clog version father spray midnight tip poem input fantasy decide buddy work strategy nest shaft
+🌍 Running a Cosmos 'pofe' app with Tendermint at http://localhost:26657.
+🌍 Running a server at http://localhost:1317 (LCD)
+
+🚀 Get started: http://localhost:12345/
+```
+
+## Creating our `Claim` type
+
+Switch back to the `pofe` directory and create a type called `claim` with the field `proof`:
+
+```
+starport type claim proof:string
+```
+
+This creates the `claim` type, as well as adding its relevant CLI commands, handlers, messages, type, queriers, and keepers.
+
+At this point, we have a working application that we can use - you can verify this by checking the output of the secondary terminal window.
+
+However, we want to modify our application so it better fits our requirements.
+
+## Modifying our application
+
+The first thing we want to do is get rid of all the `Set` actions. In reality, you could keep these, but we won't be using them in our application.
+
+Start by deleting the `poe/x/pofe/handlerMsgSetClaim.go` and `poe/x/pofe/handlerMsgSetClaim.go` files.
+
+Once this is done, you should see some errors pop up in the window that is running your applcation, saying that it can't find references to the `MsgSetClaim` struct and the `handlerMsgSetClaim` function.
+
+This can be fixed by removing functions or lines of code that implement these in the files.
```

Comment:
This is a very error-prone process. I would suggest not removing unused functionality.